modules/administration-guide/partials/proc_configuring-the-external-postgresql.adoc

```diff
@@ -0,0 +1,37 @@
+[id="configuring-the-external-postgresql_{context}"]
+= Configuring the external PostgreSQL
+
+To make workspace metadata and user information persistent, configure an external PostgreSQL database as follows:
+
+.Procedure
+
+. Define the values of the following placeholders:
++
+`_<database-user>_`:: {prod-short} server database user name
++
+`_<database-password>_`:: {prod-short} server database password
++
+`_<database>_`:: {prod-short} server database name 
+
+. Use the following SQL script to create a user and a database for the {prod-short} server to make workspace metadata persistent:
```

Comment:
One indefinite article is enough in the beginning of a list (a user and database), however it depends on the style of the whole document.

Reply:
Thanks. Fixed.